stdlib/event/player.lua

```diff
@@ -0,0 +1,80 @@
+--- Player global creation
+--  Requiring this module will automatically create a global.players[index] for all new players using stdlib's Event system
+-- @module Player
+-- @usage local Player = require 'stdlib/event/player'
+-- -- The fist time this is required it will register player creation events
+
+require('stdlib/event/event')
+require('stdlib/table')
+local fail_if_missing = require 'stdlib/core'['fail_if_missing']
+
+local Player = {}
+
+--Default data for the player
+local function new(player_index)
+    local obj = {
+        index = player_index,
+        name = game.players[player_index].name,
+    }
+    return obj
+end
+
+--Print any messages in the queue.
+local function check_message_queue(player_index)
+    if global._mess_queue then
```

Comment:
This looks like partially implemented functionality ( as in, I see no way to write to `global._mess_queue` with the stdlib). I assume you are going to build on this and document it?